book/website/community-handbook/templates/template-coworking-collabcafe.md

```diff
@@ -1,7 +1,7 @@
 (ch-template-coworking-collabcafe)=
 # Collaboration Cafe Call Template
 
-*A permanent document exists in the Framapad: [https://hackmd.io/\@KirstieJane/CollabCafe](https://hackmd.io/\@KirstieJane/CollabCafe) that is regularly updated with the empty template for the next event.*
+*A permanent document exists in the Framapad: [https://hackmd.io/@KirstieJane/CollabCafe](https://hackmd.io/@KirstieJane/CollabCafe) that is regularly updated with the empty template for the next event.*
```

Comment:
Same here, this is an older template and should be updated to https://annuel2.framapad.org/p/ttw-collaboration-cafe